test/config/hadolint.yaml

```diff
@@ -0,0 +1,17 @@
+---
+# Hadolint configuration file
+
+# configure ignore rules
+# see https://github.com/hadolint/hadolint#rules for a list of available rules.
+
+ignored: []
+failure-threshold: info
+format: tty
+no-color: false
+no-fail: false
+trustedRegistries:
```

Comment:
Currently, we are using just quay.io. Until we work on creating 'redhat-pipeline-service' org on other registries and upload our images there, I'm not confident to trust other registries. If registry.redhat.io is being used in the Openshift CI platform, then we could keep that as well.

Reply:
ok, I will delete other trusted registries.

Reply:
@bnallapeta because our Dockerfile files are using `registry.access.redhat.com/ubi8/ubi-minimal:8.6` as base image. I also need to add it as one of trusted registries.

Reply:
fixed